File: dotnet/src/MyDotey.SCF.Simple/Type/String/StringInplaceConverter.cs
using System;

namespace MyDotey.SCF.Type.String
{
    /**
     * @author koqizhao
     *
     * May 21, 2018
     */
    public class StringInplaceConverter : InplaceConverter<System.String>
    {
        public static readonly StringInplaceConverter DEFAULT = new StringInplaceConverter();
    }
}

File: java/scf-simple/src/main/java/org/mydotey/scf/type/string/StringInplaceConverter.java
package org.mydotey.scf.type.string;

import org.mydotey.scf.type.InplaceConverter;

/**
 * @author koqizhao
 *
 * May 21, 2018
 */
public class StringInplaceConverter extends InplaceConverter<String> {

    public static final StringInplaceConverter DEFAULT = new StringInplaceConverter();

    public StringInplaceConverter() {
        super(String.class);
    }

}
